keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -135,11 +137,29 @@ users:
       env:
       - name: "FOO"
         value: "bar"
+
+      # Hint to help the user install the executable. Optional.
+      installHint: "
```

Comment:
I like block quotes.
```yaml
      installHint: |
        example-client-go-exec-plugin is required to authenticate
        to the current cluster.  It can be installed:

        On macOS: brew install example-client-go-exec-plugin

        On Ubuntu: apt-get install example-client-go-exec-plugin

        On Fedora: dnf install example-client-go-exec-plugin
        ...